step-templates/liquibase-run-command.json

```diff
@@ -136,7 +136,7 @@
         "DefaultValue": "usernamepassword",
         "DisplaySettings": {
           "Octopus.ControlType": "Select",
-          "Octopus.SelectOptions": "awsiam|AWS EC2 IAM Role\nusernamepassword|Username\\Password"
+          "Octopus.SelectOptions": "awsiam|AWS EC2 IAM Role\nusernamepassword|Username\\Password\nwindowsauthentication|Windows Authentication"
```

Comment:
This new value `windowsauthentication` isnt used in the new code?
In fact, its only used in the old code.

Reply:
It is not currently, but it could be.  JDBC is a bit different, to configure the connection string for integrated, you add a component as a querystring parameter.  It's added differently depending on the database you're using.  I suppose I could add a check in the connection string builder.

Reply:
I guess the question I'd have is if someone chose that option in a step, would it cause confusion?